### DISCUSSION TITLE
GatheringInfo 스토리보드와 ViewController 가 제대로 연결이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by MARKGRAYSIRO** [DeveloperAcademy-POSTECH/MacC-GetARock#126] [오류 제보하기]
snoopy-demo 팔로업을 위해서 GatheringInfoViewController 와 스토리보드를 연결하고 있었는데, 새로운 Outlet을 생성하여 연결하려고 하니
<img width="251" alt="스크린샷 2023-01-05 오전 6 31 41" src="https://user-images.githubusercontent.com/101700487/210653752-12c5e764-e9f0-4736-976f-d8ccb930790b.png">
이와같은 에러가 발생해서 자동으로 연결이 되지 않았습니다. 이에 웹 서칭을 통해서 해결방안을 찾아서 @IBOutlet을 수동으로 입력하는 방식으로 해결했는데요, 이번에는 또 다른 문제가 나타났습니다.
[해결할때 참고한 사이트](https://macinjune.com/all-posts/web-developing/swift/xcode-swift-not-insert-new-outlet-connection-해결/)

<img width="366" alt="스크린샷 2023-01-05 오전 6 34 42" src="https://user-images.githubusercontent.com/101700487/210654248-a30f4025-b41c-4bd7-bb9c-4ebb9ca4a7e9.png">
위와같은 오류와 함께 스토리보드가 명시해둔 viewController의 @IBOutlet을 전부 찾지 못하는 현상이 발생해서, 이는 viewController의 문제라고 생각해 이것저것 확인해보았지만, 잘못된 점은 없었고, 맥북 껴켰도 여러번 해보았지만 먹히지 않아서...  좌절하고있는 상태에서...

아예 ViewController를 삭제하고 같은 이름의 ViewController를 새로 팠더니 에러가 해결되었습니다. !!(짝짝짝..)

이 문제가 제 맥북에서만 일어나고 있는 문제일 수도 있지만, 일단 제 맥북에서 확인해본 결과 develop 브랜치에 올라가있는 파일에서도 동일한 문제가 발생해서, 우선 이 문제만 수정해서 빠르게 PR을 올리려고 하는데 괜찮을까요..? ㅎㅎ

**Comment by MARKGRAYSIRO** [comment]
혹시 자신의 맥북에서도 안되는지 확인하는방법은 확인하는 방법은 develop 브랜치에서 info.plist의  스토리보드 시작점을 GatheringInfoPage로 변경해 주시면 됩니다..!!

**Comment by unuhqueen** [comment]
<img width="366" alt="스크린샷 2023-01-05 오전 6 34 42" src="https://user-images.githubusercontent.com/101700487/210654248-a30f4025-b41c-4bd7-bb9c-4ebb9ca4a7e9.png">

`GatheringInfoPage`를 시작점으로 하니까 저는 바로 이 오류가 뜨네요! hotfix PR 좋은 생각 같습니다 :)

**Comment by EuniceNam** [comment]
17bd2d5e5e7188d16cc05d327cf81277d42a7a7c 에서 pbxproj의 44, 692 줄에서 해당 파일이 빠져서 그런 것 같아요.
pbxproj의 46줄에 `66E1623F292ADFF5001F0578 /* GatheringInfoViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 66E1623E292ADFF5001F0578 /* GatheringInfoViewController.swift */; };`
670줄에 `66E1623F292ADFF5001F0578 /* GatheringInfoViewController.swift in Sources */,` 를 입력하시면 작동합니다. 
(제 거 머지하려다가 발견했습니다)

**Reply by MARKGRAYSIRO** [reply]
와.... 역시 꼼꼼하시군요.. 추가시키는게 좀 복잡해 보이는데 우선은  수정점 몇개 없는 스누피 팔로업 브랜치에 새로 파일 만들어서 빠르게 pr 올리겠습니다!!

**Reply by compuTasha** [reply]
> [17bd2d5](https://github.com/DeveloperAcademy-POSTECH/MacC-GetARock/commit/17bd2d5e5e7188d16cc05d327cf81277d42a7a7c) 에서 pbxproj의 44, 692 줄에서 해당 파일이 빠져서 그런 것 같아요.
> pbxproj의 46줄에 `66E1623F292ADFF5001F0578 /* GatheringInfoViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 66E1623E292ADFF5001F0578 /* GatheringInfoViewController.swift */; };`
> 670줄에 `66E1623F292ADFF5001F0578 /* GatheringInfoViewController.swift in Sources */,` 를 입력하시면 작동합니다.
> (제 거 머지하려다가 발견했습니다)

이거 어떻게 찾으신거에요... 최첨단 인간지능 디버거 유니스..
